setup.py

```diff
@@ -51,7 +51,7 @@
 
     # Some of the tests use Conch:
     "PyCrypto==2.6.1",
-    "pyasn1==0.1.7",
+    "pyasn1==0.1.8",
```

Comment:
Also easier to review since it's the change that means "keep using the software we've been using" instead of one that means "upgrade to some new software".